src/StoreApi/Utilities/ProductQuery.php

```diff
@@ -316,7 +316,7 @@ public function add_query_clauses( $args, $wp_query ) {
 
 		if ( $wp_query->get( 'stock_status' ) ) {
 			$args['join']   = $this->append_product_sorting_table_join( $args['join'] );
-			$args['where'] .= $wpdb->prepare( ' AND wc_product_meta_lookup.stock_status = %s ', $wp_query->get( 'stock_status' ) );
+			$args['where'] .= ' AND wc_product_meta_lookup.stock_status IN ("' . implode( '","', array_map( 'esc_sql', $wp_query->get( 'stock_status' ) ) ) . '")';
```

Comment:
👍🏻

Reply:
@mikejolley I did not find another block using this query.

Thanks for the heads-up, I've updated the docs file.